pkg/admissioncontroller/webhooks/auth/seed/authorizer_test.go

```diff
@@ -660,7 +674,7 @@ var _ = Describe("Seed", func() {
 				}
 			})
 
-			DescribeTable("should allow without consulting the graph because verb is get, list, watch, create",
+			DescribeTable("should allow without consulting the graph because verb is get, list, watch, create, delete",
```

Comment:
Why add `delete` if it's actually being removed?
```suggestion
			DescribeTable("should allow without consulting the graph because verb is get, list, watch, create",
```